### PR TITLE
translation update

### PR DESCRIPTION
**Pull request by serkan-maker**

### CODE REVIEW

File: po/tr.po [hunk 1]
```diff
@@ -412,17 +416,17 @@ msgstr "D-Bus hatası: %s"
...
-#: src/gpodder/gtkui/app.py:228
+#: src/gpodder/gtkui/app.py:227
 msgid "Bug Tracker"
 msgstr "Hata İzleyici"
 
-#: src/gpodder/gtkui/app.py:279
+#: src/gpodder/gtkui/app.py:277
 msgid "Path to gPodder home is too long"
-msgstr "gPodder eve giden yol çok uzun"
+msgstr "gPodder ana sayfasına giden yol çok uzun"
```
Comment (elelay) [inline po/tr.po:429]: not sure if it's correct: the message is used when the gpodder home folder (~/gPodder on Linux) is too deep in the folder hierarchy.
*Google translate* translates your translation back to "Path to gPodder homepage is too long", but they definitely could be wrong

**Review by elelay — COMMENTED** [review]
Thanks for the update!

**Comment by serkan-maker** [comment]
hi.  I think translation number 3 is more descriptive.. which one do you think?
1- gPodder eve giden yol çok uzun - gPodder the way home is too long
2- gPodder ana sayfasına giden yol çok uzun  -  Path to gPodder homepage is too long
3- gPodder ana klasörüne giden yol çok uzun  -  Path to gPodder home folder is too long

**Comment by auouymous** [comment]
Number three.

I know you didn't translate it, but is "Open - Aç" correct? There are other lines with aç that google says is open, but by itself google says it is hungry.

There are conflicts, did you use an older version of the file?

**Comment by serkan-maker** [comment]
google translate cannot translate some words in turkish language correctly.
sample.
I'm hungry - Karnım aç
Open the door - Kapıyı aç

I updated the old translation with the latest version pot file. POT-Creation-Date: 2021-06-10
[tr.po.tar.gz](https://github.com/gpodder/gpodder/files/6745889/tr.po.tar.gz)

**Comment by elelay** [comment]
thanks @serkan-maker!

**Comment by elelay** [comment]
merged, thanks!

**Pull request merged**